paddle/fluid/memory/allocation/retry_allocator_test.cc

```diff
@@ -93,6 +97,50 @@ TEST(RetryAllocator, RetryAllocator) {
   }
 }
 
+class DummyAllocator : public Allocator {
+ public:
+  bool IsAllocThreadSafe() const override { return true; }
+
+ protected:
+  Allocation *AllocateImpl(size_t size) override {
+    PADDLE_THROW_BAD_ALLOC("Always BadAlloc");
+  }
+
+  void FreeImpl(Allocation *) override {}
+};
+
+TEST(RetryAllocator, RetryAllocatorLastAllocFailure) {
+  {
+    RetryAllocator allocator(std::make_shared<DummyAllocator>(), 10);
+    try {
```

Comment:
Very tiny issue. You can decide whether to change or not.

Usually, actual values of variables (such as "10" here, "true/false" in some code) can decrease readability of code. A software engineer has to look at API if he/she doesn't know the second parameter of RetryAllocator is "retry_ms".

some people use
"size_t retry_ms=10; // may also be const at head of file
RetryAllocator allocator(std::make_shared<DummyAllocator>(), retry_ms)" 
or "RetryAllocator allocator(std::make_shared<DummyAllocator>(), /* retry_ms = */ 10)"

However, it depends on programmer's background knowledge whether to add it. For example, in the next line, "allocator.Allocate(100)", since "malloc" is very common in C/C++, so we don't have to add that and SWE will have sense it's byte size.

In summary, consider adding readability of code for future developer, but not too much. In this case. I suggest adding "/* retry_ms = */ 10", but keep "allocator.Allocate(100)".

Reply:
Done.